src/App.jsx

```diff
@@ -14,57 +15,72 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const defaultTitle = 'No goods selected';
+  const selectedGoodClass = 'has-background-success-light';
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood ? `${selectedGood} is selected` : defaultTitle}
```

Comment:
change syntax for better readability
```suggestion
        {
          selectedGood 
            ? `${selectedGood} is selected`
            : defaultTitle
          }
```

Reply:
Prettier is complaining and do autocorrecting when I do this because it's less than 80 digits